lib/seattleflu/id3c/api/routes.py

```diff
@@ -1,18 +1,28 @@
 import logging
-from flask import jsonify, Response, Blueprint
+from flask import jsonify, Response, Blueprint, send_file
 from flask_cors import cross_origin
 from id3c.api.routes import api_v1, blueprints
 from id3c.api.utils.routes import authenticated_datastore_session_required
+from pathlib import Path
 from . import datastore
 
 LOG = logging.getLogger(__name__)
 
+base_dir     = Path(__file__).parent.resolve()
+
 api_v2 = Blueprint('api_v2', 'api_v2', url_prefix='/v2')
 blueprints.append(api_v2)
 
 api_v3 = Blueprint('api_v3', 'api_v3', url_prefix='/v3')
 blueprints.append(api_v3)
 
+@api_v1.route("/documentation/customizations", methods = ['GET'])
```

Comment:
Could also include the `@api_unversioned.route` decorator here, since the documentation page isn't version specific? I saw that's how it's handled in id3c's API routes.

Reply:
Chatted briefly at standup, I can move this to an unversioned route.